examples/iotivity_examples/br_fw/config.h

```diff
@@ -0,0 +1,63 @@
+/*
+ // Copyright (c) 2016 Intel Corporation
+ //
+ // Licensed under the Apache License, Version 2.0 (the "License");
+ // you may not use this file except in compliance with the License.
```

Comment:
to remain clean, Apache licensed files should not be PRed directly to be located in the RIOT/example/ 
Instead, can't you pull them (+ whatever you need) along with your pkg?